test/integration/env_test.js

```diff
@@ -1,25 +1,61 @@
-suite('setting env variables', function() {
-  var co = require('co');
-  var testworker = require('../post_task');
+import assert from 'assert';
+import testworker from '../post_task';
```

Comment:
ah shoot, I put those in the wrong spot.  It works, but I didn't mean for them to be there, I'll fix that.